gemup.js

```diff
@@ -1,3 +1,4 @@
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
I think this is worth adding the dep!

Reply:
Sounds good to me! Might need to rework the README a bit to remove the instructions on using the package outside of an npm/yarn install.